docs/jsonapi/README.md

```diff
@@ -148,11 +148,13 @@ http://liiweb.test/akn/za/1993/31/eng@1993-01-31
 This example creates a new expression to the already existing work.
 
 ```
-curl -H "Content-Type: application/vnd.api+json; Accept: application/vnd.api+json" -X POST -u api_user:password --data @docs/jsonapi/create_expression.json \
-http://liiweb.test/akn/za/1993/31/eng@2019-03-11
+curl -H "Content-Type: application/vnd.api+json; Accept: application/vnd.api+json" -X POST -u admin:password --data @docs/jsonapi/create_expression.json \
+http://africanlii.local/akn/za/1993/32/eng@1993-01-31
```

Comment:
In some places we're using africanlii.local and others liiweb.test. We've been using liiweb.test so far, please can we stick to that since it's what we have setup?